### PR TITLE
Nfdiv 2334

### PR DESCRIPTION
**Pull request by velimd**
### Change description ###

Enter a description.
image
image
image
image
image

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-2334
https://tools.hmcts.net/jira/browse/NFDIV-2335
https://tools.hmcts.net/jira/browse/NFDIV-2371

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/steps/applicant1/hub-page/progress-bar.njk [hunk 1]
```diff
@@ -1,45 +1,33 @@
...
     },
     complete: displayState.isAtOrAfter('ConditionalOrderPronounced'),
-    active: displayState.state() == 'AwaitingPronouncement'
-  }, {
-    label: {
-      html: textWithVisuallyHiddenSpan(finalOrderApplication, displayState.isAtOrAfter('FinalOrderRequested'))
-    },
-    complete: displayState.isAtOrAfter('FinalOrderRequested'),
-    active: displayState.state() == 'AwaitingFinalOrder'
+    active: hasApplicantConfirmedReceipt
```
Comment (lara-mpt) [inline src/main/steps/applicant1/hub-page/progress-bar.njk:19]: won't this make it active any time after they confirm their receipt? Which doesn't seem right, as it shouldn't be active when finalOrder is

Comment (lara-mpt) [reply src/main/steps/applicant1/hub-page/progress-bar.njk:19]: I've noticed that this is how it was before, but still confused by it

File: src/main/steps/applicant1/hub-page/progress-bar.njk [hunk 2]
```diff
@@ -51,31 +39,22 @@
     active: displayState.isAtOrAfter('AwaitingAos') and displayState.isBefore('Holding')
   }, {
     label: {
-      html: textWithVisuallyHiddenSpan(conditionalOrderApplication, displayState.isAtOrAfter('AwaitingLegalAdvisorReferral'))
+      html: textWithVisuallyHiddenSpan(conditionalOrder, displayState.isAtOrAfter('AwaitingLegalAdvisorReferral'))
     },
-    complete: displayState.isAtOrAfter('AwaitingLegalAdvisorReferral'),
+    complete: displayState.isAfter('ConditionalOrderPronounced'),
     active: displayState.state() == 'AwaitingConditionalOrder' or hasApplicantConfirmedReceipt
```
Comment (lara-mpt) [inline src/main/steps/applicant1/hub-page/progress-bar.njk:45]: likewise with the above

File: src/main/steps/applicant1/hub-page/progress-bar.njk (hunk 1 above)
Comment (velimd) [reply src/main/steps/applicant1/hub-page/progress-bar.njk:19]: good point, thanks, ill have a look

File: src/main/steps/application-submitted/progress-bar.njk [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    },
+    complete: displayState.isAfter('IssuedToBailiff'),
+    active: displayState.isAtOrAfter('AwaitingAos') and displayState.isBefore('Holding'),
+    interval: weeksToResponse
+  }, {
+    label: {
+      html: textWithVisuallyHiddenSpan(conditionalOrder, displayState.isAtOrAfter('AwaitingLegalAdvisorReferral'))
+    },
+    complete: displayState.isAfter('ConditionalOrderPronounced'),
+    active: displayState.state() == 'AwaitingConditionalOrder' or hasApplicantConfirmedReceipt,
```
Comment (lara-mpt) [inline src/main/steps/application-submitted/progress-bar.njk:46]: @velimd there's a hasApplicantConfirmedReceipt here too

Comment (velimd) [reply src/main/steps/application-submitted/progress-bar.njk]: good spot, i forgot i had 2 of these files

**Review by lara-mpt — APPROVED** [review]

File: src/main/steps/applicant1/hub-page/progressBarLabels.ts [hunk 4]
```diff
@@ -24,11 +24,11 @@ const enArialLabels = {
 // @TODO translations (weeks)
```
Comment (lara-mpt) [inline src/main/steps/applicant1/hub-page/progressBarLabels.ts:24]: can remove this line

**Review by lara-mpt — APPROVED** [review]

**Pull request merged**